File: scratch.js
const fs = require('fs');
const data = JSON.parse(fs.readFileSync('./src/data/female_anatomy.json', 'utf8'));

let minX = Infinity, maxX = -Infinity;
let minY = Infinity, maxY = -Infinity;

data.back.forEach(m => {
  Object.values(m.path).flat().forEach(p => {
    const match = p.match(/m\s+([0-9.-]+),([0-9.-]+)/i);
    if (match) {
      const x = parseFloat(match[1]);
      const y = parseFloat(match[2]);
      minX = Math.min(minX, x);
      maxX = Math.max(maxX, x);
      minY = Math.min(minY, y);
      maxY = Math.max(maxY, y);
    }
  });
});

console.log('Min X:', minX, 'Max X:', maxX);
console.log('Min Y:', minY, 'Max Y:', maxY);

File: scratch.py
import json
import re

with open('./src/data/female_anatomy.json', 'r') as f:
    data = json.load(f)

min_x = float('inf')
max_x = float('-inf')
min_y = float('inf')
max_y = float('-inf')

for m in data['back']:
    for paths in m['path'].values():
        for p in paths:
            match = re.search(r'm\s+([0-9.-]+),([0-9.-]+)', p, re.IGNORECASE)
            if match:
                x, y = float(match.group(1)), float(match.group(2))
                min_x = min(min_x, x)
                max_x = max(max_x, x)
                min_y = min(min_y, y)
                max_y = max(max_y, y)

print(f"Min X: {min_x}, Max X: {max_x}")
print(f"Min Y: {min_y}, Max Y: {max_y}")
